lib/cassia/requests/turn_on_autoselect.rb

```diff
@@ -0,0 +1,36 @@
+module Cassia
+  module Requests
+    class TurnOnAutoselect
```

Comment:
can we change this to `SwitchAutoSelect` and then pass the flag in the initializer, so we have one class that can turn on or off

```ruby
def initialize(access_token: nil, flag: 1)
  @access_token = access_token
  @flag = flag
end
```